app/src/main/java/com/framgia/fsalon/screen/customer/CustomerFragment.java

```diff
@@ -15,15 +15,24 @@
 import com.framgia.fsalon.data.source.local.sharepref.SharePreferenceImp;
 import com.framgia.fsalon.data.source.remote.UserRemoteDataSource;
 import com.framgia.fsalon.databinding.FragmentCustomerBinding;
+import com.framgia.fsalon.screen.listbill.customerlist.CustomerListActivity;
 
 /**
  * Customer Screen.
  */
 public class CustomerFragment extends Fragment {
     private CustomerContract.ViewModel mViewModel;
+    private boolean mIsFilterBill;
+    private CustomerListActivity mCustomerListActivity;
 
-    public static CustomerFragment newInstance() {
-        return new CustomerFragment();
+    private CustomerFragment(boolean isFilterBill, CustomerListActivity activity) {
```

Comment:
FATAL: [ValidFragment] Avoid non-default constructors in fragments: use a default constructor plus `Fragment#setArguments(Bundle)` instead
 From the Fragment documentation:
*Every* fragment must have an empty constructor, so it can be instantiated when restoring its activity's state. It is strongly recommended that subclasses do not have other constructors with parameters, since these constructors will not be called when the fragment is re-instantiated; instead, arguments can be supplied by the caller with `setArguments(Bundle)` and later retrieved by the Fragment with `getArguments()`.